Lib/glyphsets/data.json

```diff
@@ -16432,7 +16432,8 @@
         "GF_Greek_Pro",
         "GF_Latin_Beyond",
         "GF_Latin_Vietnamese",
-        "GF_TransLatin_Arabic"
+        "GF_TransLatin_Arabic",
```

Comment:
@moyogo It looks like this is adding more data than just the PriAfrican updates here?  Can we make updates that are not PriAfrican specific in a separate PR?  I won't be the proper person to review those for inclusion here.

Reply:
This comment applies to a number of encodings in the changes to the data.json file proposed here.

Reply:
This only adds GF_Latin_PriAFrican.
JSON doesn’t have a comma on the line of the last element in a list, so adding a last element also changes the previous line.

Reply:
Got it. Sorry I misinterpreted the diff